client/src/components/NavBar.jsx

```diff
@@ -0,0 +1,47 @@
+import { BsPersonCircle, BsGeoAlt, BsTrophy } from "react-icons/bs";
+import "./NavBar.css";
+import { NavLink } from "react-router-dom";
+
+function Navbar() {
+  return (
+    <>
+      <nav className="navbar-mobile">
+        <ul className="navbar-list-mobile">
+          <li className="navbar-item">
+            <NavLink to="/position">
+              <BsGeoAlt className="navbar-link-mobile" />
+            </NavLink>
+          </li>
+          <li className="navbar-item">
+            <NavLink to="/trophy">
+              <BsTrophy className="navbar-link-mobile" />
+            </NavLink>
+          </li>
+          <li className="navbar-item">
+            <NavLink to="/profile">
+              <BsPersonCircle className="navbar-link-mobile" />
+            </NavLink>
+          </li>
+        </ul>
+      </nav>
+      <nav className="navbar-desktop">
+        <img className="logo" src="../src/assets/images/LOGO PH.jpg" alt="logo pixhunter" />
```

Comment:
It's good. But you should rename your file "LOGO PH.jpg" to "logo_ph.jpeg" or just "logo.jpeg"

Furthermore, you should go for a "png" or "favicon" instead of "jpeg"